### ISSUE TITLE
gj/gk support

### ISSUE THREAD

**Issue by rococode** [rococode/vimflowy#3]
See: WuTheFWasThat/vimflowy#62 

Thoughts from @WuTheFWasThat 

> kinda brittle, but maybe:
> 
>     add data attributes with row/column, use to get div corresponding to cursor
>     iterate through $('.cursor').siblings() either backwards/forward through $('.cursor').index(), find all things differing in offset().top and differing minimally in offset().left
>     use data attributes to know where to move cursor

**Comment by WuTheFWasThat** [comment]
excited if you could add these features!  let me know if you want to chat/have questions about stuff

**Comment by rococode** [comment]
Yeah, hoping to do this asap so I can stick some of my current written work into vimflowy haha. Probably will do some of the easier tasks to get a feel for the codebase, I'll definitely let you know if I have questions about the code/design!
